src/Sentry/Extensibility/DiagnosticLoggerExtensions.cs

```diff
@@ -211,68 +211,77 @@ public static class DiagnosticLoggerExtensions
     /// </summary>
     public static void LogError(
         this IDiagnosticLogger logger,
-        string message,
-        Exception? exception = null)
+        string message)
+        => logger.LogIfEnabled(SentryLevel.Error, null, message);
```

Comment:
Could potentially merge this with the next one, making `Exception` an optional parameter with a default value of `null`... makes for one less hop when debugging and tracing into things (and possibly one less entry on the call stack, depending on how the compiler optimises things).